addons/website_profile/static/src/scss/website_profile.scss

```diff
@@ -0,0 +1,193 @@
+                text-align: center;
+                width: 120px;
+                &:hover {
+                    border: 0px;
+                    border-radius: 0px;
+                    background-color: #875A7B;
+                }
+            }
+            .o_wprofile_navlink.active {
+                color: #212529;
+                background-color: #f2f2f2;
+                border: 0px;
+            }
+        }
+        .o_wprofile_detail {
+            .o_wprofile_content_abstract {
+                top: 1px;
+                width: 256px;
+                max-width: 256px;
+                margin-left: 15px;
+                table {
+                    width: 100%;
+                    tr {
+                        line-height: 32px;
+                    }
+                    th {
+                        font-weight: normal;
+                    }
+                }
+                .o_wprofile_rank {
+                    background-color: white;
+                }
+                .o_wprofile_abstract_info {
+                    background-color: white;
+                }
+            }
+            .o_wprofile_content_detail {
+                h4 {
+                    border-bottom: 1px solid lightgrey;
+                    color: grey;
+                    margin-bottom: 16px;
+                }
+                .o_user_badges {
+                    padding: 8px;
+                    div {
+                        padding: 8px;
+                    }
+                    img {
+                        width: 48px;
+                        height: 48px;
+                    }
+                }
+            }
+        }
+    }
+}
+
+// Rank
+.o_wprofile_ranks_presentation {
+    background-color: lightgray;
+    .o_wprofile_rank_tile {
+        width: 20%;
+        div {
+            text-align:center;
+            img {
+                height: 128px;
+            }
+        }
+    }
+}
+
+.o_wprofile_next_rank_card {
+    div {
+        text-align: center;
+        img {
+            width: 128px;
+            height: 128px;
+        }
+    }
+    border-bottom: 1px solid lightgray;
+    .o_wprofile_next_rank_progress {
+        background-color: lightgray;
+        .o_wprofile_next_rank_progress_bar {
+            background-color: #00A09D !important;
+        }
+    }
+}
+
+// Edit Profile button
+.o_wprofile_edit {
+    position: absolute;
+    top: 0px;
+    right: 0px;
+    padding-left: 16px;
+    padding-right: 16px;
+}
+
+// Badges
+.badge-gold {
+    color: $gold;
+}
+.bg-gold {
+    background-color: $gold;
+}
+
+.badge-silver {
+    color: $silver;
+}
+.bg-silver {
+    background-color: $silver;
+}
+
+.badge-bronze {
+    color: $bronze;
+}
+.bg-bronze {
+    background-color: $bronze;
+}
+
+// Other stuffs
+.o_wprofile_bb_grey {
+    border-bottom: 2px solid lightgrey;
+}
+
+.country_flag {
+    display: inline-block;
+    margin-left: 2px;
+    max-height: 13px;
+    width: auto !important;
+}
+
+.o_img_small {
```

Comment:
You should avoid having generic and not namespaced CSS. It shoudl always lie in a o_wprofile css container to be sure this css won't end up being used in other applications unless they specifically wanted it.